src/server.py

```diff
@@ -10,28 +14,82 @@
-dashboard.add_join('orders', 'customer', ['o_custkey'], ['c_custkey'])
-dashboard.add_join('partsupp', 'supplier', ['ps_suppkey'], ['s_suppkey'])
-dashboard.add_join('customer', 'nation', ['c_nationkey'], ['n_nationkey'])
-dashboard.add_join('nation', 'region', ['n_regionkey'], ['r_regionkey'])
-dashboard.add_join('lineitem', 'partsupp', ['l_suppkey', 'l_partkey'], ['ps_suppkey', 'ps_partkey'])
-dashboard.add_join('partsupp', 'part', ['ps_partkey'], ['p_partkey'])
-dashboard.register_measurement("sum", 'part', 'p_retailprice', scope=ReplicateFact('part', 'part'))
-dashboard.register_measurement("sum", 'lineitem', 'l_extendedprice * (1 - l_discount)',
-                               scope=ReplicateFact('lineitem', 'lineitem'))
+dashboard.add_join('allergies', 'patients', ['PATIENT'], ['Id']);
+dashboard.add_join('careplans', 'patients', ['PATIENT'], ['Id']);
+dashboard.add_join('procedures', 'patients', ['PATIENT'], ['Id']);
+dashboard.add_join('providers', 'organizations', ['ORGANIZATION'], ['Id']);
+dashboard.add_join('supplies', 'patients', ['PATIENT'], ['Id']);
+dashboard.add_join('observations', 'patients', ['PATIENT'], ['Id']);
+dashboard.add_join('encounters', 'patients', ['PATIENT'], ['Id']);
+dashboard.add_join('encounters', 'organizations', ['ORGANIZATION'], ['Id']);
+dashboard.add_join('encounters', 'providers', ['PROVIDER'], ['Id']);
+dashboard.add_join('encounters', 'payers', ['PAYER'], ['Id']);
+dashboard.add_join('imaging_studies', 'encounters', ['ENCOUNTER'], ['Id']);
+dashboard.add_join('allergies', 'encounters', ['ENCOUNTER'], ['Id']);
+dashboard.add_join('observations', 'encounters', ['ENCOUNTER'], ['Id']);
+dashboard.add_join('procedures', 'encounters', ['ENCOUNTER'], ['Id']);
+dashboard.add_join('careplans', 'encounters', ['ENCOUNTER'], ['Id']);
+dashboard.add_join('imaging_studies', 'patients', ['PATIENT'], ['Id']);
+dashboard.add_join('imaging_studies', 'procedures', ['PROCEDURE_CODE'], ['CODE']);
+dashboard.add_join('claims_transactions', 'providers', ['PROVIDERID'], ['Id']);
+dashboard.add_join('claims_transactions', 'patients', ['PATIENTID'], ['Id']);
+dashboard.add_join('conditions', 'encounters', ['ENCOUNTER'], ['Id']);
+dashboard.add_join('conditions', 'patients', ['PATIENT'], ['Id']);
+dashboard.add_join('devices', 'encounters', ['ENCOUNTER'], ['Id']);
+dashboard.add_join('devices', 'patients', ['PATIENT'], ['Id']);
+dashboard.add_join('immunizations', 'encounters', ['ENCOUNTER'], ['Id']);
+dashboard.add_join('immunizations', 'patients', ['PATIENT'], ['Id']);
+dashboard.add_join('medications', 'encounters', ['ENCOUNTER'], ['Id']);
+dashboard.add_join('medications', 'patients', ['PATIENT'], ['Id']);
+dashboard.add_join('medications', 'payers', ['PAYER'], ['Id']);
+dashboard.add_join('claims', 'patients', ['PATIENTID'], ['Id']);
+dashboard.add_join('claims', 'providers', ['PROVIDERID'], ['Id']);
+dashboard.add_join('payer_transitions', 'patients', ['PATIENT'], ['Id']);
```

Comment:
You could make the join graph smaller. E.g., just include two or three tables with many-to-many